frontend/src/global_styles/content/work_packages/tabs/_files.sass

```diff
@@ -132,6 +132,10 @@
       &:not(:last-child)
         margin-right: $spot-spacing-0-5
 
+  @media only screen and (max-width: 679px)
+    & &--file-list-item-floating-wrapper:hover &--file-list-item-floating-actions
```

Comment:
Just wondering: In which cases do I have a hover effect on a mobile device?

Reply:
This is probably related to the focus. Tbh I wasn't able to test it yet, a pull preview might be handy in here.

The issue is this: On iOS currently you have to click twice on the attachment. First click somehow only opens the floating actions.

On android (brave Browser) the click was working fine, but when I use the back button, the focus is still set, so I can see the floating actions, too.

Reply:
**UPDATE:** at least in the brave browser on Android, this override works fine. getting back via browser button still has the clicked file focused, but the floating actions are not visible.

Reply:
**UPDATE 2**: @apfohl tested on iOS and there the behaviour is now the same (only small difference, the focus does not render a border there, but this is unrelated).

@HDinger Do you have any further question? Or do you want to check it on your mobile once perhaps? pullpreview is available.